### ISSUE TITLE
Electronize Init SHOULD NOT depend on visual Studio file

### ISSUE THREAD

**Issue by tebeco** [ElectronNET/Electron.NET#30]
So i followed the step but never open Visual Studio, Only CLI and VsCode

Repro step :
```
$/> mkdir PocElectronNet
$/> dotnet new sln PocElectronNet
$/> cd PocElectronNet
$/PocElectronNet> mkdir src
$/PocElectronNet> cd src
$/PocElectronNet/src> mkdir PocElectronNet
$/PocElectronNet/src>cd PocElectronNet
$/PocElectronNet/src/PocElectronNet>dotnet new mvc
$/PocElectronNet/src/PocElectronNet>dotnet sln ..\..\PocElectronNet.sln add PocElectronNet.csproj
```
* Edited `Program.cs`
* Edited `Startup.cs`
* Edited `csproj`

```
$/PocElectronNet/src/PocElectronNet>dotnet restore
$/PocElectronNet/src/PocElectronNet>dotnet electronize init

Adding our config file to your project...
Search your .csproj to add the needed electron.manifest.json...
Found your .csproj: C:\Workspace\GitHub\TeBeCo\PocElectronNet\src\PocElectronNet\PocElectronNet.csproj - check for existing config or update it.
electron.manifest.json will be added to csproj.
electron.manifest.json added in csproj!
Search your .launchSettings to add our electron debug profile...

Unhandled Exception: System.AggregateException: One or more errors occurred. (Could not find a part of the path '$\PocElectronNet\src\PocElectronNet\Properties\launchSettings.json'.) ---> System.IO.DirectoryNotFoundException: Could not find a part of the path '$\PocElectronNet\src\PocElectronNet\Properties\launchSettings.json'.
```

the file not found is a file specific to Visual Studio (used by the Configuration ComboBox) allowing user to choose between IIS or Console etc ... 
It the goal of Electron is to be XPlat, and Electron.Net too, that might be a got idea to get rid of dependencies over Visual Studio generated file right ?

**Comment by tebeco** [comment]
As i saw from this code :

https://github.com/ElectronNET/Electron.NET/blob/master/ElectronNET.CLI/Commands/InitCommand.cs#L78
https://github.com/ElectronNET/Electron.NET/blob/master/ElectronNET.CLI/Commands/InitCommand.cs#L93-L109

* Create the file if it does not exists ;) ?
* Add a method for editing VsCode launch files
* Ignore the step but not throw ?

**Comment by robertmuehsig** [comment]
Ah - good catch! Sorry - I thought this file was present even when init it without Visual Studio.

First step would be to check if the file does exists do the manipulation - if not, do nothing. 

As a second step I could do the same with the VSCode launch file - but because of the nature of VSCode this might be not super important.

**Comment by robertmuehsig** [comment]
As a workaround you might create a launchSettings.json file yourself. The code is pretty stupid TBH ;)

**Comment by andymac4182** [comment]
It would be good if it created a `launchSettings.json` file so you can open it in VS later and the same debug experience will work.

**Comment by Petermarcu** [comment]
I just hit this too.

**Comment by Petermarcu** [comment]
I opened an issue against ASP.NET as well to make it so the templates create the same assets whether through VS or CLI. https://github.com/aspnet/templating/issues/142

**Comment by Petermarcu** [comment]
After creating launchSettings.json and then cleaning out the project entry and manifest from the initial failed run of `electronize init` I am past this. Here is the initial launchSettings.json in case anyone wants to copy paste.

```
{
  "iisSettings": {
    "windowsAuthentication": false,
    "anonymousAuthentication": true,
    "iisExpress": {
      "applicationUrl": "http://localhost:54587/",
      "sslPort": 0
    }
  },
  "profiles": {
    "IIS Express": {
      "commandName": "IISExpress",
      "launchBrowser": true,
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      }
    },
    "WebApplication1": {
      "commandName": "Project",
      "launchBrowser": true,
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      },
      "applicationUrl": "http://localhost:54588/"
    }
  }
}
```

**Comment by tebeco** [comment]
dunno when i’ll have time to give a shoot

**Comment by robertmuehsig** [comment]
I'm on it 👍

**Comment by robertmuehsig** [comment]
As a first step: If launchSetting is not present, do nothing.

**Comment by robertmuehsig** [comment]
The fix will be part of 0.0.6. Current approach is: We only edit the file if it is present, because creating the file itself might result in strange issues as well.

**Comment by robertmuehsig** [comment]
I released Version 0.0.6.

**Comment by tebeco** [comment]
https://github.com/aspnet/docs/issues/4021#issuecomment-342325746

is it interessting ?
